scan/git_test.go

```diff
@@ -285,8 +286,8 @@ func TestCheckoutGit(t *testing.T) {
 		// 	error: unable to index file absolutelink
 		// 	fatal: adding files failed
 		fmt.Println("Windows!!!!!!!!!!")
-		cases = append(cases, singleCase{frag: "master:absolutelink", exp: "FROM scratch" + eol + "EXPOSE 5000", fail: false})
-		cases = append(cases, singleCase{frag: "master:parentlink", exp: "FROM scratch" + eol + "EXPOSE 5000", fail: false})
+		cases = append(cases, singleCase{frag: "default:absolutelink", exp: "FROM scratch" + eol + "EXPOSE 5000", fail: false})
```

Comment:
@xyxyxyxyxyxy  fyi - there are some test failure related to the branch validation change.